### PR TITLE
[DEV-1618] Import webinar subscriptions to ddb script

### PR DESCRIPTION
**Pull request by christian-calabrese**
#### List of Changes

Implemented script and tests to allow import of users' webinar subscriptions from Amazon Cognito attributes to DynamoDB.

#### Motivation and Context

Writing to Amazon Cognito user's attributes will be deprecated in favor of DynamoDB for enhanced query capabilities and near to limitless scalability. This script allows to import existing subscriptions from Cognito to DynamoDB.

#### How Has This Been Tested?

The script has been tested via unit tests. Moreover, it has been run several times writing to a local dynamodb container and to the development account DynamoDB table.

#### Types of changes

- [ ] Chore (nothing changes by a user perspective)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by tommaso1 — CHANGES_REQUESTED** [review]
Ok to me, it's just missing the changeset

**Comment by christian-calabrese** [comment]
> Ok to me, it's just missing the changeset

Thank you @tommaso1!
I thought about creating a changeset or not. However, since it's not an actual change to the software and I couldn't find a module to locate it to, I didn't create it.

What do you think?

**Comment by tommaso1** [comment]
I would put it on the general webside project, just to track on release that we developed this script, but I get your point

**Review by tommaso1 — APPROVED** [review]

File: scripts/webinar-subscriptions-import/src/README.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+1. Ensure you have Python installed on your system. You can download it from [python.org](https://www.python.org/).
+
+2. Navigate to the project directory:
+
+```cd scripts/webinar-subscriptions-import```
+
+3. Install the necessary dependencies:
+
+```pip3 install -r requirements.txt```
```
[the remark this answers is not in the document]
Comment (christian-calabrese) [reply scripts/webinar-subscriptions-import/src/README.md:17]: Yes, pip comes with python as the default package installer

**Review by datalek — APPROVED** [review]
My knowledge of Python is a bit rusty, but seems fine to me!

**Pull request merged**